build.gradle.kts

```diff
@@ -53,7 +53,7 @@ spotless {
     lineEndings = com.diffplug.spotless.LineEnding.UNIX
     kotlin {
         targetExclude("**/*.gradle.kts", "**/build/generated-sources/**/*.kt")
-        ktlint().editorConfigOverride(
+        ktlint(libs.versions.ktlint.get()).editorConfigOverride(
```

Comment:
added explicit version so spotless updates don't involve formatting changes, ktlint upgrades should be formatting the code